features/telemetry/protocols/netflow/parser/src/main/java/org/opennms/netmgt/telemetry/protocols/netflow/parser/session/UdpSessionManager.java

```diff
@@ -48,13 +49,13 @@
 import org.opennms.netmgt.telemetry.protocols.netflow.parser.state.ExporterState;
 import org.opennms.netmgt.telemetry.protocols.netflow.parser.state.OptionState;
 import org.opennms.netmgt.telemetry.protocols.netflow.parser.state.ParserState;
+import org.opennms.netmgt.telemetry.protocols.netflow.parser.state.TemplateState;
 
 import com.google.common.collect.Iterables;
 import com.google.common.collect.Maps;
-import org.opennms.netmgt.telemetry.protocols.netflow.parser.state.TemplateState;
 
 public class UdpSessionManager {
-    final Map<TemplateKey, TimeWrapper<TemplateOptions>> templates = Maps.newConcurrentMap();
+    final ConcurrentMap<TemplateKey, TimeWrapper<TemplateOptions>> templates = Maps.newConcurrentMap();
```

Comment:
Has this change a particular reason or is it just to emphasize that concurrency is impotent?

Reply:
The method `ConcurrentMap.compute` that is used when updating the template is defined on `ConcurrentMap`.